lib/node_modules/@stdlib/math/base/special/bernoullif/lib/bernoullif.json

```diff
@@ -0,0 +1,34 @@
+[
+  1.00000000000000000000000000000000000000000,
```

Comment:
These values all have very high precision. Is there a reason for that?

Reply:
No, I have used similar values as of `math/base/special/bernoulli`. Should I make it less precise?